snorkeling_helper/label_functions/disease_gene/disease_associates_gene.py

```diff
@@ -0,0 +1,1164 @@
+# Global Imports for Positive (1) Negative (0) and ABSTAIN (-1)
+from ..global_helpers import (
+    ABSTAIN,
+    NEGATIVE,
+    POSITIVE,
+    OrderedDict,
+    pathlib,
+    re,
+    pd,
+    ltp,
+    labeling_function,
+    get_tagged_text,
+    get_tokens_between,
+    get_token_windows
+)
+import numpy as np
+
+"""
+DISTANT SUPERVISION
+"""
+path = (
+    pathlib.Path(__file__)
+    .joinpath("../../knowledge_bases/disease_associates_gene.tsv.xz")
+    .resolve()
+)
+pair_df = pd.read_csv(path, dtype={"sources": str}, sep="\t")
+knowledge_base = set()
+for row in pair_df.itertuples():
+    if not row.sources or pd.isnull(row.sources):
+        continue
+    for source in row.sources.split("|"):
+        key = str(row.entrez_gene_id), row.doid_id, source
+        knowledge_base.add(key)
+
+
+@labeling_function()
+def LF_HETNET_DISEASES(c):
```

Comment:
Are these label functions the same as previously?

Reply:
>Are these label functions the same as previously?

Yes these notebooks are the same as the previous notebook; however, one catch here is that they have been debugged and updated to snorkel's most current version. I just wanted to separate the previous repository with this new repository, so people have a new start if they want to begin from scratch.